packages/models-library/src/models_library/api_schemas_webserver/projects.py

```diff
@@ -78,14 +78,18 @@ class ProjectGet(OutputSchema):
     ui: EmptyModel | StudyUI | None = None
     quality: dict[str, Any] = {}
     dev: dict | None = None
-    permalink: ProjectPermalink = FieldNotRequired()
-    workspace_id: WorkspaceID | None
-    folder_id: FolderID | None
+    permalink: ProjectPermalink | None = FieldNotRequired()
+    workspace_id: WorkspaceID | None = None
```

Comment:
@matusdrobuliak66 confirmed it is optional

Reply:
these are only for jsonschema generation right? what confuses me is when you say "optional" this does not mean with a default of None